src/components/VCSMenu.tsx

```diff
@@ -337,20 +437,112 @@ export class VCSMenu extends React.Component<VCSMenuProps, VCSMenuState> {
       selection.forEach(variableName => {
         plotString += variableName + ", ";
       });
-      plotString += `${gm}, ${temp})`;
+      plotString += `${temp}, ${gm})`;
+      console.log("plotString:", plotString)
       this.props.inject(plotString);
     }
   }
 
-  save(): void {
-    let response: string | null = prompt(
-      "Please enter the name to save the plot as."
-    );
-    if (response == null || response == "") {
-      notebook_utils.showMessage("Notice", "Injection cancelled.");
-    } else {
-      this.props.inject(`canvas.png('${response}')`);
+  async save() {
+    let plotName = this.state.plotName
+    if(plotName == null || plotName == ""){
+      this.setState({ validateExportName : true })
+      return
+    }
+    else{
+      this.setState({ validateExportName : false })
+    }
+    let fileFormat = this.state.plotFileFormat
+    console.log("fileFormat: ", fileFormat)
+    if(fileFormat == null || fileFormat == ""){
+      this.setState({ validateFileFormat : true })
+      return
     }
+    else {
+      this.setState({ validateFileFormat : false })
+    }
+
+    let capture = null
+    if(this.state.captureProvenance){
+      capture = 1
+    }
+    else{
+      capture = 0
+    }
+    console.log("capture:", capture)
+
+    console.log("height in save:", this.state.height)
+    console.log("width in save:", this.state.width)
+    console.log("plotUnits in save:", this.state.plotUnits)
+
+    if(fileFormat === "png"){
```

Comment:
Change to: if(fileFormat == "png"), to match the others. Unless there's a reason for it to need '===' ?

Reply:
I used triple equals for strict equivalence checking